internal/cephfs/snapshot.go

```diff
@@ -25,6 +25,10 @@ import (
 	"github.com/golang/protobuf/ptypes/timestamp"
 )
 
+// autoProtect points to the snapshot auto-protect feature of
+// the subvolume.
+const autoProtect = "snapshot-autoprotect"
```

Comment:
```suggestion
const (
           autoProtect = "snapshot-autoprotect"
          )
```
lets add `()` so that we can add more consts later

Reply:
Thanks, Will change it.